eng/pipelines/templates/jobs/sign-and-pack.yml

```diff
@@ -26,24 +26,7 @@ jobs:
     parameters:
       SigningKeyCode: 'CP-230012' # MS Cert
       BinaryPath: $(Build.ArtifactStagingDirectory)/signed
-      BinaryPattern: |
-        **/windows-*/**/azmcp.dll
-        **/windows-*/**/azmcp.exe
-        **/windows-*/**/mcptmp.dll
-        **/windows-*/**/mcptmp.exe
-        **/windows-*/**/*.Mcp.*.dll
-
-  - template: pipelines/steps/binary-signing.yml@azure-sdk-build-tools
-    parameters:
-      SigningKeyCode: 'CP-231522' # MS 3rd Party Cert
-      BinaryPath: $(Build.ArtifactStagingDirectory)/signed
-      BinaryPattern: |
-        **/windows-*/**/ModelContextProtocol*.dll
-        **/windows-*/**/Newtonsoft.Json.dll
-        **/windows-*/**/Npgsql.dll
-        **/windows-*/**/OpenAI.dll
-        **/windows-*/**/OpenTelemetry*.dll
-        **/windows-*/**/YamlDotNet.dll
+      BinaryPattern: $(WindowsSigningPatterns)
```

Comment:
Looks like you moved this into its own PR https://github.com/microsoft/mcp/pull/885